### ISSUE TITLE
Cannot launch Android examples

### ISSUE THREAD

**Issue by cactuschibre** [react-native-maps/react-native-maps#3933]
# Bug report

## Summary

When launching the set of examples in /example directory, it crashes on `npx react-native run-android`.

## Environment info

- Master branch of react-native-maps (commit 907be3238d731aabbd2b02d1f469d6c7fe31807d)
- `npx react-native info`:

```bash
> npx react-native info
info Fetching system and libraries information...
System:
    OS: Linux 5.10 Kali GNU/Linux Rolling 2021.2
    CPU: (8) x64 Intel(R) Core(TM) i5-8300H CPU @ 2.30GHz
    Memory: 386.02 MB / 15.49 GB
    Shell: 5.1.8 - /bin/bash
  Binaries:
    Node: 14.17.2 - ~/.nvm/versions/node/v14.17.2/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v14.17.2/bin/yarn
    npm: 6.14.7 - ~/.nvm/versions/node/v14.17.2/bin/npm
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: 4.0 AI-193.6911.18.40.6626763
  Languages:
    Java: 11.0.12 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.64.1 => 0.64.1 
  npmGlobalPackages:
    *react-native*: Not Found
```

## Steps to reproduce

```bash
cd example/
npm install
npx react-native run-android
```

**Describe what you expected to happen:**

The set of examples for Android launches.

**Describe what you actually happens:**

```bash
/home/seb/projects/react-native-maps/example/android/app/src/debug/AndroidManifest.xml Error:
	uses-sdk:minSdkVersion 16 cannot be smaller than version 21 declared in library [com.facebook.react:react-native:0.64.1] /home/seb/.gradle/caches/transforms-2/files-2.1/155fd089da50ef28f6c11e8639b531f2/jetified-react-native-0.64.1/AndroidManifest.xml as the library might be using APIs not available in 16
	Suggestion: use a compatible library with a minSdk of at most 16,
		or increase this project's minSdk version to at least 21,
		or use tools:overrideLibrary="com.facebook.react" to force usage (may lead to runtime failures)
```

## Reproducible sample code

N/A

**Comment by christopherdro** [comment]
Did you try bumping up your project's minSDK ?

**Comment by MaRaSu** [comment]
I noticed the same and simply bumped up minSDK in example/android/build.gradle file. Then built successfully in Android Studio.

@christopherdro I wonder how up-to-date are all the build settings in the RNM project? When I forked the repo for developing the caching versions of URLTile + WMSTile I had to tweak quite a few settings for XCode & Android Studio. Not sure which ones were absolutely needed as that was my first native code project and my tweaking of settings was mostly trial-n-error rather than based on solid understanding of the build system on either platform.
